docs/managing-compute-resources.asciidoc

```diff
@@ -25,7 +25,14 @@ For Elasticsearch objects, make sure to consider the heap size when you set reso
 ----
 spec:
   nodeSets:
-  - podTemplate:
+  - name: elasticsearch
+    count: 1
+    config:
+      node.master: true
+      node.data: true
+      node.ingest: true
+      node.store.allow_mmap: false
```

Comment:
The config seems a bit irrelevant here, would it be readable enough if we remove it? (we still have name, count and podTemplate at that level).